ext/intl/locale/locale_methods.c

```diff
@@ -105,7 +199,7 @@ static int16_t findOffset(const char* const* list, const char* key)
 {
 	const char* const* anchor = list;
 	while (*list != NULL) {
-		if (strcmp(key, *list) == 0) {
+		if (strcasecmp(key, *list) == 0) {
```

Comment:
We weren't doing a case-insensitive comparison, but language tags are case-insensitive.